react-forms/src/Select.tsx

```diff
@@ -88,7 +88,7 @@ export default function Select<T>(props: IProps<T>) {
         value={GetRecordValue()}
         disabled={props.Disabled == null ? false : props.Disabled}
       >
-        {props.EmptyOption ? <option value="">{props.EmptyLabel !== undefined? props.EmptyLabel : ''}</option> : null}
+        {props.EmptyOption === undefined ? <option value="">{props.EmptyLabel !== undefined? props.EmptyLabel : ''}</option> : null}
```

Comment:
```suggestion
        {(props.EmptyOption ?? true) ? <option value="">{props.EmptyLabel ?? ''}</option> : null}
```
This is actually a bug otherwhise